dashboard/lib/api/v1/pd/workshop_score_summarizer.rb

```diff
@@ -94,8 +94,15 @@ def get_score_for_workshops(workshops, facilitator_breakdown: false, include_fre
       responses = PEGASUS_DB[:forms].where(source_id: enrollment_ids, kind: 'PdWorkshopSurvey')
 
       if facilitator_breakdown
+        # We need to figure out how many responses each facilitator got - the below two lines
+        # return a histogram showing FacilitatorName=># Responses.
+        # Using 'how_often_given_feedback_s' for no particular reason - any of the facilitator
+        # specific responses would do fine here
+        facilitator_name_responses = responses.map {|x| JSON.parse x[:data]}.map {|x| x['how_often_given_feedback_s']}.flat_map(&:keys)
+        facilitator_name_frequency = Hash[*facilitator_name_responses.group_by {|v| v}.flat_map {|k, v| [k, v.size]}]
```

Comment:
How about breaking this out into its own function, with a unit test? Something like `calculate_facilitator_name_frequencies(responses)`

I'd prefer more descriptive variables in the blocks on line 101, i.e. `response` and `data` respectively